Polyfills/AbortController/Source/AbortController.cpp

```diff
@@ -26,8 +26,12 @@ namespace Babylon::Polyfills::Internal
 
     void AbortController::Abort(const Napi::CallbackInfo&)
     {
-        m_signal.Set("aborted", true);
-        m_signal.Get("onabort").As<Napi::Function>().Call({});
+        AbortSignal* sig = AbortSignal::Unwrap(m_signal.Value());
+
+        if (sig != nullptr)
+        {
+            sig->Abort();
+        }
```

Comment:
do we want to throw an error if `sig == nullptr`?

Reply:
Doesn't seem like it's possible for the user to set the signal as null, so the only way for that to happen is for napi to fail the construction of it. For extra safety I could add throws for this and inside of the controller construction.

Reply:
If it's not supposed to be possible, I would recommend replacing the if/throw with an assert.